### PR TITLE
TER-87 delete rows - supports whereFilters

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: TER-87
Connection: https://github.com/keboola/connection/pull/4141

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

**Review by zajca — APPROVED** [review]
Cajk, povolil sem BQ testy předpokládám, že to merdžnem spolu s BQ

File: tests/Backend/CommonPart2/SlicedImportsTest.php [hunk 1]
```diff
@@ -62,7 +62,6 @@ public function testSlicedImportSingleFile(): void
     {
         $this->skipTestForBackend([
             self::BACKEND_TERADATA,
-            self::BACKEND_BIGQUERY,
```
Comment (jirkasemmler) [inline tests/Backend/CommonPart2/SlicedImportsTest.php:65]: toto ale bude pravda az s updatem driveru ne? a to v tom PR connectino se nedeje

**Review by jirkasemmler — CHANGES_REQUESTED** [review]

File: tests/Backend/CommonPart2/SlicedImportsTest.php (hunk 1 above)
Comment (zajca) [reply tests/Backend/CommonPart2/SlicedImportsTest.php:65]: to píšu v mojém revjů :)

Comment (zajca) [reply tests/Backend/CommonPart2/SlicedImportsTest.php:65]: imo @martinjunger referencuje špatné PR v connection ten multiple test pojede až s https://github.com/keboola/connection/pull/4148

Comment (zajca) [reply tests/Backend/CommonPart2/SlicedImportsTest.php:65]: a jo on tu multiple test není :facepalm: pletu to já :)

Comment (zajca) [reply tests/Backend/CommonPart2/SlicedImportsTest.php]: smazal sem svůj commit

Comment (martinjunger) [reply tests/Backend/CommonPart2/SlicedImportsTest.php]: jo, sorry za zmatky, ale tady jsem resil primarne aby ty testy jeli s novyma parametrama - skutecny povoleni testu je az tady https://github.com/keboola/storage-api-php-client/pull/972

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**